Resources/AzureDevOps/JobTemplates/RunIndividualTest.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
+  RequiresAzurePowershell: false
+  RequiresPowershellCore: false
+  AzureSubscription: ''  
+  TestType: ''  # One of 'Acceptance', 'Quality', "Unit"
+  ResultsFileSuffix: ''
+  CodeCoveragePercentage: 80
+
+jobs:
+- job: ${{ parameters.TestName }}
+  displayName: 'Run ${{ parameters.TestType }} tests'
+  steps:
+  - task: DeleteFiles@1
+    displayName: 'Remove old test and code coverage files'
+    inputs:
+      SourceFolder: '$(System.DefaultWorkingDirectory)/Tests'
+      Contents: |
+        **/TEST-${{ parameters.ResultsFileSuffix }}.xml
+        **/CODECOVERAGE-${{ parameters.ResultsFileSuffix }}.xml
+
+  - ${{ if eq(parameters.RequiresAzurePowershell, true) }}:
```

Comment:
Not familiar with this syntax, is it the same as setting a condition property

Reply:
Yeah, it's an interesting concept. 
You remember the ` - ${{ each <property> }}: `  syntax I used in dfc-servicetaxonomy-apifunction a while back?
This is the conditional execution version of that

Basically,  if the property evaluates to true at parse-time, it will add everything that is 'underneath' it into the resulting template, otherwise it'll exclude it. 

It's pretty neat for dropping tasks in and out of execution scope,  which is what I'm using it for